static/docs/commands-reference/install.md

```diff
@@ -51,6 +51,8 @@ The installed Git hook automates executing `dvc push`.
 - A `pre-push` hook executes `dvc push` before `git push` to upload files and
   directories under DVC control to remote storage.
 
+_If a hook already exists, DVC will raise exception._
```

Comment:
_an_ exception - I think 

Clarify a bit what should I do if I see this exception. Edit manually? Can we put a link to Git docs at least so that people can start from there?